PWGHF/TableProducer/treeCreatorD0ToKPi.cxx

```diff
@@ -354,11 +386,14 @@ struct HfTreeCreatorD0ToKPi {
       double yD = yD0(candidate);
       double eD = eD0(candidate);
       double ctD = ctD0(candidate);
+      int candType = 0;
       if (candidate.isSelD0()) {
-        fillTable(candidate, prong0, prong1, 0, invMassD0ToPiK(candidate), cosThetaStarD0(candidate), ctD, yD, eD, 0, 0);
+        candType |= o2::aod::full::SelD0;
+        fillTable(candidate, prong0, prong1, 0, candType, invMassD0ToPiK(candidate), cosThetaStarD0(candidate), ctD, yD, eD, 0, 0);
       }
       if (candidate.isSelD0bar()) {
-        fillTable(candidate, prong0, prong1, 1, invMassD0barToKPi(candidate), cosThetaStarD0bar(candidate), ctD, yD, eD, 0, 0);
+        candType |= o2::aod::full::SelD0bar;
+        fillTable(candidate, prong0, prong1, 1, candType, invMassD0barToKPi(candidate), cosThetaStarD0bar(candidate), ctD, yD, eD, 0, 0);
```

Comment:
In the `isSelD0bar` block, `candType` can already have the `isSelD0` bit set which you probably do not want. Therefore you should assign the value with `=` instead of adding the bit.
You don't even need to define the `candType` variable. You can just pass the value directly to the `fillTable` function.